src/main/java/com/pnudev/springonlinedelivery/User.java

```diff
@@ -0,0 +1,26 @@
+package com.pnudev.springonlinedelivery;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Table
+@Entity
+public class User {
+	@Id
+	public long id;
+	public String first_name;
```

Comment:
You should have some kind of validation for those fields. Theoretically user can input whatever information they want but it won't be a valid name/email/mobile. Think about including some kind of validation for entity fields.

Reply:
Done